packages/core-mobile/app/new/routes/(signedIn)/(modals)/accountSettings/manageNetworks/addCustomNetwork.tsx

```diff
@@ -0,0 +1,6 @@
+import { AddEditNetworkScreen } from 'features/accountSettings/screens/manageNetworks/AddEditNetworkScreen'
+import React from 'react'
+
+export default function AddCustomNetwork(): JSX.Element {
+  return <AddEditNetworkScreen mode="create" />
+}
```

Comment:
```suggestion
export { AddEditNetworkScreen as default } from 'features/accountSettings/screens/manageNetworks/AddEditNetworkScreen'
```
looks like mode prop is not really there